lib/index.js

```diff
@@ -3,7 +3,7 @@ var clone = require("clone");
 var Joi = require("joi");
 var url_template = require("url-template");
 
-var s3_host = url_template.parse("https://{bucket}.s3.dualstack.{region}.amazonaws.com");
+var s3_host = url_template.parse("https://{bucket}.s3.amazonaws.com");
```

Comment:
Are you sure this is a correct change? I know that S3 buckets in us-east-1 can be referenced like this, but buckets in other regions can't

Reply:
I used it for `eu-west-3` for personal purposes, worked fine. As written in s3 pages, "
S3 does not require the selection of a region.".

![fireshot capture 4 - s3 management console_ - https___s3 console aws amazon com_s](https://user-images.githubusercontent.com/2424954/47099566-d9901c00-d235-11e8-8935-4cfa7389d2a5.png)